### ISSUE TITLE
Custom theme

### ISSUE THREAD

**Issue by TalAmuyal** [extr0py/oni#259]
# TL;DR

Themes that aren't the defaultly supplied, don't work.

# Details

In my regular Neovim work-flow I use the Solaraized (dark) color-scheme.
For some reason, it doesn't work well on Oni (that is, it is very similar to deleting the `colorscheme onedark` part of the default configuration of Oni).

When trying to use some of Neovim's default color schemes on Oni, I understood that most of them (or all?) have little to no effect.

Does a color scheme for Oni need to be specialized in any way?
Does anyone knows what I'm talking about or have a suggestion?

A side-by-side example is attached.
![screenshot from 2017-02-26 09-20-23](https://cloud.githubusercontent.com/assets/5097613/23342444/0893c5ba-fc63-11e6-998f-1e2ee3722da2.png)

In each window, the Solaraized color-scheme is set.

**Comment by bert88sta** [comment]
I use salaried dark, I'd suggest using nvm true color or term gui colors.
I'll post my vimrc later if you can't get this resolved

**Comment by TalAmuyal** [comment]
This far, I've tried `set termguicolors` but didn't see any change.
Would be nice if you could dig up the missing piece.

**Comment by metalelf0** [comment]
Also make sure you have the following lines in your `~/.oni/config.json` file:

```
    "prototype.editor.backgroundOpacity": 1,
    "prototype.editor.backgroundImageUrl": null,
    "prototype.editor.backgroundImageSize": null
```

This disables the default background image oni provides.

**Comment by extr0py** [comment]
Hmm, in theory it should work - but it looks like somehow `solarized` isn't being picked up by the neovim instance that we're using in Oni.

One question - where are you saving `solarized`? Are you saving it in your `nvim\colors` directory or are you using a plugin manager and saving it elsewhere?

In addition, the output of `:set runtimepath` in both the Oni and Neovim case would be helpful - we can compare and see if there is a runtimepath not being set when Oni is running, and then troubleshoot from there.

**Comment by bert88sta** [comment]
give this a try

frankier/neovim-colors-solarized-truecolor-only

(Plugin manager of your choice)

along with 

let $NVIM_TUI_ENABLE_TRUE_COLOR=1

and 

set termguicolors


It works for me, ket me know how this goes

**Comment by extr0py** [comment]
Sounds like this has been addressed by PR #256 , so I'll close. Feel free to reopen if this wasn't fully addressed
